src/node_v8.cc

```diff
@@ -295,6 +295,154 @@ static void SetHeapStatistics(JSONWriter* writer, Isolate* isolate) {
   writer->json_arrayend();
 }
 
+static v8::Local<v8::Object> ConvertHeapStatsToJSObject(
+    v8::Isolate* isolate, const cppgc::HeapStatistics& stats) {
+  Local<Context> context = isolate->GetCurrentContext();
+  // Space Statistics
+  v8::LocalVector<v8::Value> spaceStatisticsArray(isolate);
+  for (size_t i = 0; i < stats.space_stats.size(); i++) {
+    const cppgc::HeapStatistics::SpaceStatistics& spaceStats =
+        stats.space_stats[i];
+    // Page Statistics
+    v8::LocalVector<v8::Value> pageStatisticsArray(isolate);
+    for (size_t j = 0; j < spaceStats.page_stats.size(); j++) {
+      const cppgc::HeapStatistics::PageStatistics& pageStats =
+          spaceStats.page_stats[j];
+      // Object Statistics
+      v8::LocalVector<v8::Value> objectStatisticsArray(isolate);
+      for (size_t k = 0; k < pageStats.object_statistics.size(); k++) {
+        const cppgc::HeapStatistics::ObjectStatsEntry& objectStats =
+            pageStats.object_statistics[k];
+        Local<v8::Name> objectStatsNames[] = {
+            objectStatsNames[0] =
+                FIXED_ONE_BYTE_STRING(isolate, "allocatedBytes"),
+            objectStatsNames[1] =
+                FIXED_ONE_BYTE_STRING(isolate, "objectCount")};
```

Comment:
I think you can just do this:

```suggestion
            FIXED_ONE_BYTE_STRING(isolate, "allocatedBytes"),
            FIXED_ONE_BYTE_STRING(isolate, "objectCount")
          };
```

it's similar for the values below.